.github/workflows/gke.yaml

```diff
@@ -4,26 +4,43 @@ on:
   pull_request_target:
     types:
       - "labeled"
-  #pull_request: #uncomment for testing
+  # WARNING FOR REVIEWERS
+  # The `pull_request` webhook event should never be uncommented on `master`,
+  # so as not to allow access to or leak repo's secrets.
+  # Please be extra careful when reviewing changes to GitHub workflows.
+  # pull_request:
   #  types:
   #    - "labeled"
   push:
     branches:
       - master
 
 env:
-  clusterName: cilium-cli-ci-${{ github.run_number }}
+  clusterName: cilium-cli-ci-${{ github.run_id }}
   zone: us-west2-a
+  check_name: ${{ github.workflow }}
```

Comment:
I think we should change this workflow name to something that won't cause the action check status to be confused with actual run status (set by this step).

I would set `env.check_name` to "ConformanceGKE` and set top level `name` to something like `prepare GKE run` or something that will not be alerting if it's skipped.

Reply:
Fixed!